include/cupla/cudaToCupla/runtime.hpp

```diff
@@ -77,3 +77,10 @@
 #define make_cudaPos(...) make_cuplaPos(__VA_ARGS__)
 
 #define make_cudaPitchedPtr(...) make_cuplaPitchedPtr(__VA_ARGS__)
+
+//math
+#define __fdividef(a,b) ((a)/(b))
```

Comment:
please move this changes to a separate pull request, after I merged the pull you can rebase this pull request.